src/chrome/content/rules/Officeworks.com.au.xml

```diff
@@ -0,0 +1,67 @@
+<!--
+	For other Coles coverage, see Coles.com.au.xml
+
+
+	Non-functional subdomains:
+		- autodiscover	(r)
+		- av	(t)
+		- blog	(m)
+		- careers	(m)
+		- catalogues	(m)
+		- forestsurvey	(SSL connection error)
+		- uat.forestsurvey	(SSL connection error)
+		- isc	(t)
+		- lyncwebs	(t)
+		- m	(m)
+		- ovtrnexpe001	(t)
+		- photos	(t)
+		- www.photos	(t)
+		- pimtest	(t)
+		- print	(t)
+		- t.service	(m)
+		- ugc	(m)
+		- vpn	(t)
+		- webconf	(t)
+
+	e: expired certificate
+	h: http redirect
+	i: invalid certificate chain
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Officeworks">
+
+	<target host="officeworks.com.au" />
+	<target host="www.officeworks.com.au" />
+	<target host="api.officeworks.com.au" />
+	<target host="as2test.officeworks.com.au" />
+	<target host="t.comms.officeworks.com.au" />
+	<target host="disney.officeworks.com.au" />
+	<target host="edi.officeworks.com.au" />
+	<target host="enterpriseregistration.officeworks.com.au" />
+	<target host="idfed.officeworks.com.au" />
+	<target host="jabber.officeworks.com.au" />
+	<target host="mail.officeworks.com.au" />
+	<target host="onet.officeworks.com.au" />
+	<target host="onet-admin.officeworks.com.au" />
+	<target host="ovtllasa001.officeworks.com.au" />
+	<target host="ovtllexpe001.officeworks.com.au" />
+	<target host="ovtllexpe002.officeworks.com.au" />
+	<target host="ovtrnasa001.officeworks.com.au" />
+	<target host="ovtrnexpe002.officeworks.com.au" />
+	<target host="owprod-bigidea.officeworks.com.au" />
+	<target host="pim.officeworks.com.au" />
```

Comment:
`printandcopy.officeworks.com.au`

Reply:
`pbsactivate.officeworks.com.au`

Reply:
`printandcopy` 404 for me, I can't find a sample link for `<test url>`.

Reply:
`http://printandcopy.officeworks.com.au/OPSWebshopv2/flyers/create/create.seam?productname=place-cards&isSecure=true`

Reply:
`&` should be `&amp;`